methcomp/blandaltman.py

```diff
@@ -181,74 +182,77 @@ def plot(
         pkws.update(point_kws or {})
 
         # Get parameters
-        mean, mean_ci = self.result["mean"], self.result["mean_CI"]
+        mean, mean_CI = self.result["mean"], self.result["mean_CI"]
+        loa_upper, loa_upper_CI = self.result["loa_upper"], self.result["loa_upper_CI"]
+        loa_lower, loa_lower_CI = self.result["loa_lower"], self.result["loa_lower_CI"]
+        sd_diff = self.result["sd_diff"]
 
         # individual points
         ax.scatter(self.mean, self.diff, **pkws)
 
         # mean difference and SD lines
         ax.axhline(mean, color=color_mean, linestyle="-")
-        ax.axhline(mean + self.loa_sd, color=color_loa, linestyle="--")
-        ax.axhline(mean - self.loa_sd, color=color_loa, linestyle="--")
+        ax.axhline(loa_upper, color=color_loa, linestyle="--")
+        ax.axhline(loa_lower, color=color_loa, linestyle="--")
 
         if reference:
             ax.axhline(0, color="grey", linestyle="-", alpha=0.4)
 
         # confidence intervals (if requested)
         if self.CI is not None:
-            ax.axhspan(self.CI_mean[0], self.CI_mean[1], color=color_mean, alpha=0.2)
-            ax.axhspan(self.CI_upper[0], self.CI_upper[1], color=color_loa, alpha=0.2)
-            ax.axhspan(self.CI_lower[0], self.CI_lower[1], color=color_loa, alpha=0.2)
+            ax.axhspan(*mean_CI, color=color_mean, alpha=0.2)
```

Comment:
should let this alpha be `self.CI_alpha`, initialized by default to `0.2` in constructor

Reply:
added this now as a plot() argument since it's only graphically relevant